src/cmd/main.go

```diff
@@ -17,6 +18,18 @@ func main() {
 
 	slog.Info("Hello World")
 
+	for _, partnerId := range config.KnownPartnerIds {
+		partnerConfig, err := config.NewConfig(partnerId)
+		if err != nil {
+			// TODO - add an ADR talking about this. We're not crashing if a single config doesn't load in case only one partner is impacted
+			slog.Error("Unable to load or parse config", slog.Any(utils.ErrorKey, err), slog.String("partner", partnerId))
+		}
+
+		config.Configs[partnerId] = partnerConfig
+		slog.Info("config found", slog.String("Id", partnerId))
+
+	}
```

Comment:
This code looks like `config` package initialization code.  You can actually put this into the `config` package directly by using an `init()` function.

In `config/config.go` (or really any file in the `config` package), you'd create...

```go
func init() {
    // the logic outlined in this function
}
```

You can read a bit more about the execution order of the `init` function in StackOverflow.  [Answer one](https://stackoverflow.com/a/24790378/969310), and [answer two](https://stackoverflow.com/a/49831018/969310).

Reply:
Would having it in the init function cause the settings to be reloaded if the config package got imported into another package? 

We put the code in the main function thinking it was the cleanest and safest way to load the settings once and start up  and have them there for reference everywhere else.

Reply:
An `init` function is only executed once (the first time the package is imported).